### ISSUE TITLE
Add matrix!

### ISSUE THREAD

**Issue by ajohnson23** [cjbarber/hipchat-alternatives#16]
Add matrix + vector.im! :)

https://github.com/matrix-org/synapse#introduction
https://github.com/vector-im/vector-web
https://vector.im

**Comment by hrj** [comment]
:+1: 

Matrix is open, decentralised, with multiple server and client implementations.

Note for those who are not familiar with it: vector.im is a web-client for the matrix.org protocol.

**Comment by MurzNN** [comment]
Yes, Matrix is very good free opensource alternative to HipChat, also it have bridges to HipChat rooms: via [Matterbridge](https://github.com/42wim/matterbridge) so we can keep talking with HipChat users via Matrix client.

**Comment by izzy** [comment]
Matrix also has a giant advantage: It's now backed by the French government because they want to use it internally.
